cli/doc/ts_type.rs

```diff
@@ -69,6 +69,21 @@ impl Into<TsTypeDef> for &TsLitType {
           boolean: None,
         },
       ),
+      TsLit::Tpl(tpl) => {
+        // A template literal in a type is not allowed to have
+        // expressions, so there will only be one quasi.
```

Comment:
I'm not sure if quasi is the singular form of quasis... I'm not so familiar with the word quasis other than seeing it in the babel ast and in swc.